templates/base/head.tmpl

```diff
@@ -78,14 +78,14 @@
 				MaxTimeout: {{NotificationSettings.MaxTimeout}},
 				EventSourceUpdateTime: {{NotificationSettings.EventSourceUpdateTime}},
 			},
-      {{if .RequireTribute}}
+{{if .RequireTribute}}
```

Comment:
Has eslint forced you to move this to column 0?

That could be irritating

Reply:
It's not directly related here, I just noticed the `if` was intended with spaces but the `end` with tabs which was a bit distracting and it seemed better to just unintend that block.

Reply:
Undid it.

Reply:
Oh I was just worried that eslint had made you change it!